### DISCUSSION TITLE
Something went wrong.

### DISCUSSION THREAD

**Discussion by SingleSourceStudios** [AntlerVC/firetable#1301] [Support / Q&A] [answered]
ROWY TRACING ID: ROWYERR-0BFA-7B40-1CA8-3E2BCCE6678D

Please assist me here.

<img width="593" alt="Screenshot 2023-06-17 at 04 43 41" src="https://github.com/rowyio/rowy/assets/99079017/d264e6a3-caca-454a-83d0-60a3d35b3451">

**Comment by BeeBombshell** [comment, marked as answer]
Hey @SingleSourceStudios, did you just set up your Rowy Account? We're currently investigating this issue... Till then could you try the following:

- Manually deleting the account ending in `.rowy` in Firebase Authentication, this will trigger a new user account creation. Hopefully, this would resolve your issue. 
![image](https://github.com/rowyio/rowy/assets/53828745/9cb3b788-5fa9-42b5-a4ca-3f1d9422c49c)

- Could you check if the service account `rowy-service@[projectid].iam.gserviceaccount.com` has at least these roles on the GCP IAM page?
![image](https://github.com/rowyio/rowy/assets/53828745/18724d24-655a-4901-888b-b82e20a3b0ef)

A few users came up with this issue, but it got fixed for them automatically after a while... Let us know if it gets fixed for you as well.

**Comment by SingleSourceStudios** [comment]
Hi,

Thank you for the response, it wss just a novice error that I made and it is solved now thank you, amazing platform.



Rainier Potgieter
Director | Single Source Studios (Pty) Ltd
 | +27828548448 <tel:%7C++27828548448> | ***@***.*** ***@***.***>
Address: 
Creative Offices
Umhlanga, Durban
·
Studio A - 34 Ashfield Ave, Umgeni Business Park, Durban 
| ***@***.*** ***@***.***>
·
Studio B - Remhoogte Farm, Church Road, Riebeeck, Capetown
| ***@***.*** ***@***.***>
· 
Website: |  <>www.singlesource.co.za <http://www.singlesource.co.za/> 
 <https://www.facebook.com/singlesourcestudios>  <https://www.linkedin.com/in/single-source-studios/>  <https://instagram.com/singlesourcestudios>  <https://youtube.com/channel/channel/UCJjE9LGQfu7ayimYMlIIc6A>  <https://behance.net/rainierpotgiet1>  <https://wa.me/+27828548448>

 


Confidentiality Clause

Privileged/Confidential information may be contained in this message.
If you are not the addressee indicated in this message (or responsible for delivery of the message to such person),
you may not copy or deliver this message to anyone. In such case, you should destroy this message, and notify us immediately.
